### PR TITLE
🤖 Simplify workflows

### PR DESCRIPTION
**Pull request by arpanlaha**
Simplifies GitHub Actions workflows as follows:
- Disables `Emojify PR Title` by default
- Uses `working-directory` defaults as appropriate
- Prefixes job names with workflow names to avoid collisions on required checks

### CODE REVIEW

**Review by n3a9 — APPROVED** [review]
sigh

File: .github/workflows/emojify-pr.yml [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
 
-on: pull_request
+# delete lines 5-8 and uncomment line 11 to to enable the Emojify PR Title workflow
+
+on:
+  pull_request:
+    branches-ignore:
+      - "**"
+
+# on: pull_request
```
Comment (yousefa00) [inline .github/workflows/emojify-pr.yml:10]: Is it line 10 or line 11? on github preview, it shows line 10

**Review by yousefa00 — APPROVED** [review]
😎

File: .github/workflows/emojify-pr.yml (hunk 1 above)
Comment (arpanlaha) [reply .github/workflows/emojify-pr.yml:10]: ope yes ty

**Pull request merged**